src/httpd_jsonapi.c

```diff
@@ -1827,16 +1803,99 @@ jsonapi_reply_queue_tracks_add(struct httpd_request *hreq)
       if (pos >= 0)
 	pos += count;
 
-      ttl_count += count;
+      *total_count += count;
     }
   while ((uri = strtok(NULL, ",")));
```

Comment:
small Q - shouldn't this be `strtok_r()` to avoid potential data-race across threads?  I see `strtok` being used in `http.c` whilst other files are using `strtok_r`